<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fun37td</class>
 <widget class="QDialog" name="fun37td">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {	
	border-image: url(:/rc/res/main_bk.png) 0 00 0 0 repeat;
}
#widget_2{ 
border-image: url(:/rc/res/main_bk.png) 50 00 0 0 repeat;
}
QPushButton {
	border: 1px solid  rgb(0, 0, 0);
	border-radius: 0px;
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0.505682 rgba(35, 97, 222, 247), stop:0.539773 rgba(61, 114, 223, 247), stop:1 rgba(93, 142, 239, 238));
	color: rgb(255, 255, 255);
	font: 16px;
}
QGroupBox::title {
  color: rgb(255, 255, 255);
  subcontrol-origin: margin;
  subcontrol-position: top center;
  padding-top: 10px;  
  border-top: 1px solid rgb(255, 255, 255);
  font:bold 14px;
}
QGroupBox {
	border: 1px solid rgb(255, 255, 255); 
    border-radius: 9px;
	padding:2px; font: bold 16px;	
	background-color: rgba(199, 199, 199, 100);
}

QTableWidget::item {
   
}
QHeaderView::section {
    background-color:  rgb(221, 221, 221); 
	font: 18px;
	border: 0px solid #6c6c6c;
	height: 25px;
}

QTableWidget {
border-left: 1px solid rgb(255, 255, 255); 
border-right: 1px solid rgb(255, 255, 255); 
background-color: transparent;
color: rgb(255, 255, 255);
font: bold 14px;
border-bottom-left-radius:10px;
border-bottom-right-radius:10px;
}</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>800</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="btItemNext">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>350</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>下一件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btPageNext">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btMoveDown">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>下移</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btPageUp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-left-radius:10px;
border-bottom-left-radius:10px;</string>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btExit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>350</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btMoveUp">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-left-radius:10px;
border-bottom-left-radius:10px;</string>
    </property>
    <property name="text">
     <string>上移</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btSave">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>350</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-left-radius:10px;
border-bottom-left-radius:10px;</string>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>471</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>工单信息</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_info">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>471</width>
       <height>291</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="rowCount">
      <number>9</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>97</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>24</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>新建行</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>新建列</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>新建列</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>工单号</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>工作中心</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>总件数</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string>产品编号</string>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string>产品描述</string>
      </property>
     </item>
     <item row="5" column="0">
      <property name="text">
       <string>标准周期</string>
      </property>
     </item>
     <item row="6" column="0">
      <property name="text">
       <string>本件模穴数</string>
      </property>
     </item>
     <item row="7" column="0">
      <property name="text">
       <string>次品总数</string>
      </property>
     </item>
     <item row="8" column="0">
      <property name="text">
       <string>生产总数</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="btItemUp">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>350</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top-left-radius:10px;
border-bottom-left-radius:10px;</string>
    </property>
    <property name="text">
     <string>上一件</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>派工单列表</string>
    </property>
    <widget class="QTableWidget" name="tableWidget_Order">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>291</width>
       <height>291</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>110</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <column/>
     <column/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
